firstvoices/backend/tests/test_resources/test_dictionary_resources.py

```diff
@@ -37,7 +37,7 @@ def build_table(data: list[str]):
         headers = [
             # these headers should match what is produced by fv-nuxeo-export tool
```

Comment:
Since we're keeping the resources (with modifications) and tests, some of these comments should be removed.

If I can help with building automated test data, lmk.

Reply:
Yes, I have updated this comment, and I will be reaching out to you for help in building the test data once the dry-run base infra is complete on my end so we can test it better and catch all the edge cases. Thank you :)